packages/lb-utils/src/annotation/ResourceManager.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * ResourceManager.
+ *
+ * FeatureList:
+ * 1. Preload resource. (Include img.)
+ *
+ * TODO:
+ * 1. Preload Data by resource size.
+ *
+ */
+
+const DEFAULT_CACHE_SIZE = 2;
+
+class ResourceManager {
+  public cacheMap: Map<string, boolean>;
+  public size: number;
+
+  constructor({ size = DEFAULT_CACHE_SIZE }: { size?: number } = { size: DEFAULT_CACHE_SIZE }) {
+    this.cacheMap = new Map();
+    this.size = size;
+  }
+
+  public fetchResource(url: string) {
+    if (this.cacheMap.has(url)) {
+      return;
+    }
+
+    fetch(url).then(() => {
+      this.cacheMap.set(url, true);
+    });
+  }
+
+  public preloadResourceList(urlList: string[]) {
+    urlList.forEach((url) => {
+      this.fetchResource(url);
+    });
+  }
+
+  public quickPreload({
```

Comment:
why this fn has no comment and reference

Reply:
Becase it is just to quickPreload